File: PRAK204-2510817120013-SalmaNurLathifah.py
pi = 3.142857

nilai = input().split()

if len(nilai) == 1:
    r = float(nilai[0])
    t = float(input())
else:
    r, t = map(float, nilai)

volume = pi * r * r * t
luas = 2 * pi * r * (r + t)
keliling = 2 * pi * r

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: PRAK204-2510817120013-Salma-Nur-Lathifah.c
#include <stdio.h>

int main()
{
    float r, tinggi, volume, luas, keliling, pi = 3.142857;

    scanf ("%f %f", &r, &tinggi);

    volume = pi * (r * r) * tinggi;
    luas = 2 * pi * r * (r + tinggi);
    keliling = 2 * pi * r;

    printf ("Volume = %.2f \n", volume);
    printf ("Luas = %.2f \n", luas);
    printf ("Keliling = %.2f \n", keliling);

    return 0;
}